types/apidoc/apidoc-tests.ts

```diff
@@ -12,14 +12,13 @@ const apidocOutput = createDoc({
     verbose: true,
     single: true,
     debug: true,
-    parse: true,
     colorize: true,
     filters: { aFilter: '' },
     languages: { aLanguage: '' },
     parsers: { aParser: '' },
     workers: { aWorker: '' },
     silent: true,
```

Comment:
```suggestion
    quiet: true,
```
according to this
https://github.com/apidoc/apidoc/blob/master/CHANGELOG.md?plain=1#L16-L21